Cargo.toml

```diff
@@ -9,6 +9,7 @@ keywords = ["no_std", "ethereum"]
 edition = "2018"
 
 [dependencies]
+log = "0.4"
```

Comment:
Should have `default-features = false` and enable `log/std` when `std` feature is enabled.

Reply:
Sorry, I missed that.
It is fixed